test/FlatFeeCalculator/FlatFeeCalculator.t.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-FileCopyrightText: 2024 Toucan Protocol
+//
+// SPDX-License-Identifier: UNLICENSED
+
+// If you encounter a vulnerability or an issue, please contact <info@neutralx.com>
+pragma solidity ^0.8.13;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {FeeCalculator} from "../../src/FeeCalculator.sol";
+import {FeeDistribution} from "../../src/interfaces/IFeeCalculator.sol";
+import {FlatFeeCalculator} from "../../src/FlatFeeCalculator.sol";
+
+contract FlatFeeCalculatorTest is Test {
```

Comment:
This test suite looks like it's a fuzz test suite. Maybe rename the directory and filename from `FlatFeeCalculator/FlatFeeCalculator.t.sol` to `FlatFeeCalculatorFuzzy/FlatFeeCalculator.fuzzy.t.sol` and the contract name to:
```suggestion
contract FlatFeeCalculatorTestFuzzy is Test {
```
to stay consistent with the rest of the tests

Reply:
Done